tutorial/1. user_tutorial.ipynb

```diff
@@ -0,0 +1,665 @@
+   "metadata": {},
+   "source": [
+    "The <i>define_reaction()</i> function takes in the gas and species variables returned from the <i>define_system()</i> function.</br>\n",
+    "Using the species and gas variables, the function creates two reactions: arr1 and arr2.</br>\n",
+    "The arr1 variable represents the conversion of the X reacant into the Y product with some pre-defined constants as well.</br>\n",
+    "The arr2 variable is just like arr1, but instead it represents the conversion of the Y reactant into the Z product.</br>\n",
+    "These reactions are then bundled into a dictionary called rxns just like the species before, and it is returned.</br>\n",
+    "<b>Note:</b> MusicBox allows for users to experiment with an array of reaction types.</br>\n",
+    "Go [here](https://ncar.github.io/musica/api/python.html#module-musica.mechanism_configuration) to view a list of supported reactions and their parameters."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f465f101",
+   "metadata": {},
+   "source": [
+    "## 3. Defining Mechanisms\n",
+    "\n",
+    "A mechanism represents a set of species with their respective phases and reactions.</br>\n",
+    "For extended documentation about the Mechanism class, go [here](https://ncar.github.io/musica/api/python.html#musica.mechanism_configuration.Mechanism).</br>\n",
+    "To create a mechanism in MusicBox:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "25da888c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def define_mechanism(gas, species, rxns): # Create the mechanism that is defined by the species, phases, and reactions\n",
+    "    mechanism = mc.Mechanism(name=\"tutorial_mechanism\", species=list(species.values()), phases=[gas], reactions=list(rxns.values()))\n",
+    "    return mechanism"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b4ee16f5",
+   "metadata": {},
+   "source": [
+    "The <i>define_mechanism()</i> function builds upon the previous two functions, taking in the previously-returned gas, species, and rxn as parameters.</br>\n",
+    "This function simply creates a mechanism variable that represents an instance of the Mechanism class.</br>\n",
+    "The mechanism is first given a name, then the species, gas, and rxns are passed into their respective parameters.</br>\n",
+    "It is then returned from the function."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6cbf0090",
+   "metadata": {},
+   "source": [
+    "## 4. Creating a Box Model\n",
+    "\n",
+    "Box models allow you to encase your previously-created mechanism and add conditions that can change the reactions' rates over time.</br>\n",
+    "The box model is defined in the MusicBox class, which an instance of must be created so that you can access the functionality of it.</br>\n",
+    "You can also define the length of the simulations and the time steps.</br>\n",
+    "To initialize a new box model:"
```

Comment:
```suggestion
    "Box models allow you solve your previously-created mechanism under conditions that can change the reactions' rates over time.</br>\n",
    "The box model is defined in the MusicBox class, which an instance of must be created so that you can access the functionality of it.</br>\n",
    "You can also define the length of the simulations and the time steps.</br>\n",
    "To initialize a new box model:"
```